sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -83,8 +88,8 @@ OrchestrationResponse filter(@Nonnull @PathVariable(name = "level") final Sensit
     final var filter = new AzureContentFilter().hate(level);
     final var prompt =
         new OrchestrationPrompt(
-                "This prompt demonstrates how to hit the fucking input filter. And hit it hard, like we mean it.")
-            .withInputContentFilter(filter);
+            "This prompt demonstrates how to hit the fucking input filter. And hit it hard, like we mean it.",
+            config.withInputContentFilter(filter));
```

Comment:
I'm not sure whether a "prompt" should know the orchestration config, or should the prompt be a message.

How about the following?
```
// update config
config.withInputContentFilter(filter));
// init a prompt, being a message
final var prompt = new OrchestrationPrompt("my prompt");
// send a request with a prompt + a config
return client.chatCompletion(prompt, config);
```